domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/caserole/CaseRoleEntityFieldValueValidatorImplTest.java

```diff
@@ -0,0 +1,164 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.caserole;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseRoleEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
+
+import static org.hamcrest.core.StringContains.containsString;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("CaseRoleEntity Field Value Validator Tests")
+class CaseRoleEntityFieldValueValidatorImplTest {
+    private CaseRoleEntity caseRoleEntity;
+    private CaseRoleEntityValidationContext caseRoleEntityValidationContext;
+
+    @InjectMocks
+    private CaseRoleEntityFieldValueValidatorImpl classUnderTest;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+        classUnderTest = new CaseRoleEntityFieldValueValidatorImpl();
```

Comment:
This line can be removed as instantiation is already achieved by decorator `@InjectMocks`

Reply:
done